browse/services/document/author_affil.py

```diff
@@ -64,8 +64,7 @@ def parse_author_affil(authors: str) -> List[List[str]]:
 
 
 def _parse_author_affil_split(author_line: str) -> Dict:
-    """
-    Split author line into author and affiliation data.
+    """Split author line into author and affiliation data.
```

Comment:
Hmm, thought we got rid of this; should be using https://github.com/arXiv/arxiv-base/blob/master/arxiv/util/authors.py

Reply:
If the arxiv-base master branch is using it, then it ought to be in develop already

Reply:
Deleted author_affil.py.